idp-service/src/test/java/io/mosip/idp/services/AuthorizationServiceTest.java

```diff
@@ -268,23 +303,23 @@ public void getOauthDetails_withValidAcrInDBAndValidAcrInReq() throws Exception
         clientDetail.setId("34567");
         clientDetail.setRedirectUris("https://localshot:3044/logo.png,http://localhost:8088/v1/idp,/v1/idp");
         clientDetail.setClaims("email,given_name");
-        clientDetail.setAcrValues("level1,level2,level5");
+        clientDetail.setAcrValues("mosip:idp:acr:generated-code,mosip:idp:acr:linked-wallet");
```

Comment:
why acr values is comma separated and not array of string ?

Reply:
As we save acrs, claims, and redirect URIs as comma-separated values in DB, we are mocking it the same way in this test case.